tests/jobQueue/v1/getBulkUpsertStatus.js

```diff
@@ -171,13 +173,13 @@ describe('tests/jobQueue/v1/getBulkUpsertStatus.js, ' +
     });
   });
 
-  it('Even when hundreds of upserts fail, the valid upsert should ' +
+  it('Even when lots of upserts fail, the valid upsert should ' +
```

Comment:
yes is it the same for most part. Only change is in the number of samples upserted.